micro-deps/micro-deps/src/test/groovy/com/ofg/infrastructure/discovery/MicroserviceConfiguration.groovy

```diff
@@ -2,6 +2,23 @@ package com.ofg.infrastructure.discovery
 
 class MicroserviceConfiguration {
 
+    public static final String LOAD_BALANCING_DEPENDENCIES = """
+                            {
+                                "pl": {
+                                    "this": "com/ofg/service",
+                                    "dependencies": {
+                                        "ping" : {
+                                            "path": "com/ofg/ping",
+                                            "load-balancer": "sticky"
```

Comment:
Can you put an example of `load-balancer` option to https://github.com/4finance/micro-infra-spring/blob/master/micro-deps/micro-deps/README.md?